keylime/tagging.py

```diff
@@ -0,0 +1,196 @@
+'''
```

Comment:
This is a bit nit picky (naming things usually is), but I think this module should be named something like `keylime.failure` rather than `keylime.tagging`. Yes we are tagging failures, but it's mainly about failures. Or do you envision tagging other events besides just failures?

Reply:
Makes sense. I'll rename it. I think that all events that we tag are failures. The non security relevant parts are part of the normal logging.